### PR TITLE
Initial

### PR DESCRIPTION
**Pull request by hjain-git**
Pull request for review and merge in develop branch.

### CODE REVIEW

File: wdio.conf.js [hunk 1]
```diff
@@ -0,0 +1,286 @@
...
+  // =====================
+  // Server Configurations
+  // =====================
+  // Host address of the running Selenium server. This information is usually obsolete as
+  // WebdriverIO automatically connects to localhost. Also, if you are using one of the
+  // supported cloud services like Sauce Labs, Browserstack, Testing Bot or LambdaTest you don't
+  // need to define host and port information because WebdriverIO can figure that out
+  // according to your user and key information. However, if you are using a private Selenium
+  // backend you should define the host address, port, and path here.
+  //
```
Comment (SujaySawant) [inline wdio.conf.js]: Remove commented lines.

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,5 @@
+node_modules
+ORG-wdio.conf.js
+sample.png
+ios.wdio.conf.js
+android.wdio.conf.js
```
Comment (SujaySawant) [inline .gitignore]: Include all the lines from [this](https://github.com/browserstack/browserstack-examples-reusable-artifacts/blob/main/gitignores/.gitignores.js) gitignore file mentioned in our reusable artefacts repo into your gitignore file.

**Review by SujaySawant — CHANGES_REQUESTED** [review]
Please make the necessary changes.

File: wdio.conf.js (hunk 1 above)
Comment (hjain-git) [reply wdio.conf.js]: done

File: .gitignore (hunk 2 above)
Comment (hjain-git) [reply .gitignore]: done

**Pull request merged**